### ISSUE TITLE
Port to Swift 3

### ISSUE THREAD

**Issue by gui-dos** [sveinbjornt/STPrivilegedTask#13]
I tried to rewrite the core part of STPrivilegedTask in Swift 3 and it is working on macOS Sierra:

https://github.com/gui-dos/Guigna/blob/9fdd75ca0337c8081e2a2727960389c7dbf8d694/Legacy/Guigna-Swift/Guigna/GAgent.swift#L42-L80

**Comment by Maxatma** [comment]
actually, it is swift 2, isn't it? I have `'init' is unavailable: use 'withMemoryRebound(to:capacity:_)' to temporarily view memory as another layout-compatible type.` this error

**Comment by terkelg** [comment]
I get that too @Maxatma. Did you find a solution?

**Comment by Maxatma** [comment]
@terkelg no, I didn't. I just have found that I no need this. All I was need is put the real path for shell command.
`        shell("/usr/local/bin/mytool", "somestuff i do with that tool ")
`
When I used this helper:
```
//MARK: HELPERS

@discardableResult
func shell(_ args: String...) -> Int32 {
    
    let task        = Process()
    task.launchPath = "/usr/bin/env"
    task.arguments  = args
    task.launch()
    task.waitUntilExit()
    return task.terminationStatus
}
```
for command-line tool - it work well if u use
`        shell( "somestuff i do with needed tool ")

But if I use same for mac-os app, I get some kind of 'permission denied' error. 
It fixed if u put the real path for tool.

**Comment by x13a** [comment]
Hello from 2021) You are welcome [swift-5](https://github.com/x13a/authorization-swift/blob/master/Sources/Authorization/Authorization.swift).
